sources/packages/backend/apps/db-migrations/src/sql/StudentAssessments/Add-reported-date-col.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+  sims.student_assessments
+ADD
+  COLUMN reported_date TIMESTAMPTZ;
+
+COMMENT ON COLUMN sims.student_assessments.reported_date IS 'Timestamp for the last reported assessment id.';
```

Comment:
Please avoid technical terms like timestamp in comment. This column indicates the date at which the assessment causing the study date change has been reported.